### ISSUE TITLE
Argument comments alignment uses spaces

### ISSUE THREAD

**Issue by revolter** [onevcat/VVDocumenter-Xcode#97]
It would be great if the alignment could be switched from settings to use tabs instead of spaces.

```
/**
 *	(tab)Does stuff
 *
 *	(tab)@param labels   (spaces)The labels.
 *	(tab)@param selector (space)The method.
 */
```

**Comment by onevcat** [comment]
Currently the tab and space is mixed used when you uncheck the "use space instead of tab" option.

I will take a look at it soon to see what I can do later.

**Comment by revolter** [comment]
Ok, I updated the comment to fix an error and add an example to make it more clear.

**Comment by onevcat** [comment]
Should be fixed in #123
